src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -16,6 +16,10 @@
     <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">dotnet5.4</PackageTargetFramework>
     <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetGroup)' == 'netcore50' and '$(ProjectJson)' == '' ">
+    <ProjectJson>netcore50\project.json</ProjectJson>
```

Comment:
That is OK. the netcore50 will be an entirely different dependency group so it won't need to have any mention of Csp. It is just a convenience to avoid having multiple project.json files. The only reason you would need to have multiple project.json files is if you need to have multiple configurations with the same TFM that have differing dependencies. I suspect that is the case for win\project.json, but I suspect that isn't true for the netcore50\project.json.